apps/server/src/modules/tool/service/tool-validation.service.spec.ts

```diff
@@ -105,6 +105,26 @@ describe('ToolValidation', () => {
 				await expect(result).resolves.not.toThrow();
 			});
 
+			it('should validate the default with the regular expression', async () => {
+				const externalToolDO: ExternalToolDO = externalToolDOFactory
+					.withCustomParameters(1, { default: 'test', regex: '[t]', regexComment: 'mockComment' })
+					.buildWithId();
+
+				const result: Promise<void> = service.validateCreate(externalToolDO);
+
+				await expect(result).resolves.not.toThrow();
+			});
+
+			it('should not validate the default with the regular expression', async () => {
+				const externalToolDO: ExternalToolDO = externalToolDOFactory
+					.withCustomParameters(1, { default: 'es', regex: '[t]', regexComment: 'mockComment' })
+					.buildWithId();
+
+				const result: Promise<void> = service.validateCreate(externalToolDO);
+
+				await expect(result).rejects.toThrow();
```

Comment:
Please do not use only toThrow. Use the exact message to be sure to expect the correct exception